private/buf/bufcli/errors.go

```diff
@@ -114,7 +114,7 @@ func NewOrganizationNameAlreadyExistsError(name string) error {
 // NewRepositoryNameAlreadyExistsError informs the user that a repository
 // with that name already exists.
 func NewRepositoryNameAlreadyExistsError(name string) error {
-	return fmt.Errorf("a repository named %q already exists", name)
+	return fmt.Errorf("a module named %q already exists", name)
```

Comment:
Wondering if `a module with fully qualified name <FQN> ...` might be more appropriate here (but could also be more than a surface level rename 😅 ).